testsuite/features/build_validation/add_non_MU_repositories/centos7_minion_add_iso.feature

```diff
@@ -5,7 +5,7 @@
 Feature: Add the CentOS 7 distribution custom repositories
 
   Scenario: Download the iso of CentOS 7 DVD and mount it on the server
-    When I mount as "centos-7-iso" the ISO from "http://mirror.chpc.utah.edu/pub/centos/7/isos/x86_64/CentOS-7-x86_64-DVD-2009.iso" in the server
+    When I mount as "centos-7-iso" the ISO from "http://mirror.chpc.utah.edu/pub/centos/7/isos/x86_64/CentOS-7-x86_64-DVD-2009.iso" having an expected SHA256 checksum of "e33d7b1ea7a9e2f38c8f693215dd85254c3a4fe446f93f563279715b68d07987" in the server
```

Comment:
I think that providing the whole checksum value in the scenario is to technical, and we try to avoid too much technical detail on Gherkin side.

Instead, I would have something like:
`When I mount as "centos-7-iso" the ISO from "http://mirror.chpc.utah.edu/pub/centos/7/isos/x86_64/CentOS-7-x86_64-DVD-2009.iso" in the server, validating its checksum``

And inside the step you do something like you did here:

```
  # Calculate the SHA256 checksum of the downloaded ISO and compare it with the expected one
  iso_checksum = get_target('server').run("sha256sum #{iso_path} | awk '{print $1}'").strip
  raise "SHA256 checksum verification failed" unless downloaded_checksum == checksum
```

But you obtain the value of `checksum` variable from `https://mirror.chpc.utah.edu/pub/centos/7/isos/x86_64/sha256sum.txt` or the `.sha256` file if exists.

Reply:
That would also be my suggestion.